docs/tutorials/kafka.ipynb

```diff
@@ -0,0 +1,1332 @@
+        "\n",
+        "This kind of a functionality has limited use cases during the training phase wherein, once a datapoint has been consumed by the model it is no longer required and can be discarded.\n",
+        "\n",
+        "**However, this functionality shines when it comes to robust inference with exactly-once semantics.**"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "95Chcbd9xThl",
+        "colab_type": "text"
+      },
+      "source": [
+        "#### Track the offset lag of the `testcg` consumer group w.r.t `susy-test` topic"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "9uz3km0RxUG7",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 105
+        },
+        "outputId": "c39dbe3d-4382-4013-9d19-4e80f7712e0a"
+      },
+      "source": [
+        "!confluent-5.4.1/bin/kafka-consumer-groups --bootstrap-server 127.0.0.1:9092 --describe --group testcg\n"
+      ],
+      "execution_count": 24,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "\n",
+            "GROUP           TOPIC           PARTITION  CURRENT-OFFSET  LOG-END-OFFSET  LAG             CONSUMER-ID                                  HOST            CLIENT-ID\n",
+            "testcg          susy-test       0          21833           21833           0               rdkafka-0556cae8-c449-48af-9c2b-77224c3919b5 /172.28.0.2     rdkafka\n",
+            "testcg          susy-test       1          18167           18167           0               rdkafka-0556cae8-c449-48af-9c2b-77224c3919b5 /172.28.0.2     rdkafka\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "I8Wg0_eXMKL9",
+        "colab_type": "text"
+      },
+      "source": [
+        "Once the `current-offset` matches the `log-end-offset` for all the partitions, it indicates that the consumers have fetched all the messages from the kafka topic."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "P8QAS_3k1y3u",
+        "colab_type": "text"
+      },
+      "source": [
+        "### References:\n",
+        "\n",
+        "- Baldi, P., P. Sadowski, and D. Whiteson. “Searching for Exotic Particles in High-energy Physics with Deep Learning.” Nature Communications 5 (July 2, 2014)\n",
+        "\n",
+        "- SUSY Dataset: https://archive.ics.uci.edu/ml/datasets/SUSY#\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "FONyXboRYe51",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        ""
+      ],
```

Comment:
Can you also remove this section? It seems this section is empty.

Reply:
removed in the new commit